lib/tool_mount_point/wpcc.rb

```diff
@@ -0,0 +1,6 @@
+module ToolMountPoint
+  class Wpcc < Base
+    EN_ID = 'wpcc'
```

Comment:
The 8460 user story is on the PR description and the detailed information about the acceptance criteria too. Is there anything on PR description that is not clear the intent of this changes?